suseconnect-ng.spec

```diff
@@ -131,10 +131,62 @@ install -D -m 644 %_builddir/go/src/%import_path/man/SUSEConnect.8 %buildroot/%_
 install -D -m 644 %_builddir/go/src/%import_path/man/zypper-migration.8 %buildroot/%_mandir/man8/zypper-migration.8
 install -D -m 644 %_builddir/go/src/%import_path/man/zypper-search-packages.8 %buildroot/%_mandir/man8/zypper-search-packages.8
 
+# Install the SUSEConnect --keepalive timer and service.
+install -D -m 644 %_builddir/go/src/%import_path/suseconnect-keepalive.timer %buildroot/%_unitdir/suseconnect-keepalive.timer
+install -D -m 644 %_builddir/go/src/%import_path/suseconnect-keepalive.service %buildroot/%_unitdir/suseconnect-keepalive.service
+ln -sf service %buildroot/%_sbindir/rcsuseconnect-keepalive
+
 find %_builddir/..
 # we currently do not ship the source for any go module
 rm -rf %buildroot/usr/share/go
 
+%pre
+%service_add_pre suseconnect-keepalive.service suseconnect-keepalive.timer
+
+# in pre blocks the old version is still installed. This way we can detect
+# if --keepalive was already present before
+kainfo=0
```

Comment:
this part is different than the original because it needs to handle various upgrade scenarios (e.g. SUSEConnect -> suseconnect-ng).